runtime/onert/core/include/util/ShapeInference.h

```diff
@@ -123,6 +123,8 @@ class StaticInferer : public ir::OperationVisitor
   // TODO write op starting from M
   void visit(const ir::operation::Neg &op);
   // TODO write op starting from P
+  // TODO write op starting from N
```

Comment:
@hasw7569 Please remove comment `... from P` (since `Pow` is added) and `...from N` (beyond this PR).

Reply:
modified : )